Tests/test_file_mic.py

```diff
@@ -1,10 +1,46 @@
-from helper import unittest, PillowTestCase
+from helper import unittest, PillowTestCase, hopper
 
-from PIL import MicImagePlugin
+from PIL import Image, MicImagePlugin
+
+TEST_FILE = "Tests/images/hopper.mic"
 
 
 class TestFileMic(PillowTestCase):
 
+    def test_sanity(self):
+        im = Image.open(TEST_FILE)
+        im.load()
+        self.assertEqual(im.mode, "RGBA")
+        self.assertEqual(im.size, (128, 128))
+        self.assertEqual(im.format, "MIC")
+
+        im2 = hopper("RGBA")
+        self.assert_image_similar(im, im2, 123.5)
```

Comment:
There certainly appear to be gamma or colorspace issues. I'd almost believe sRGB/naive colorspace. But that's not enough to account for the epsilon. You might want to look at the alpha, it's not going to show up in `im.show()` but if it's 0 vs 255, then that would account for enough of the rest.